odl/solvers/advanced/proximal_operators.py

```diff
@@ -369,15 +730,57 @@ def _call(self, x, out):
                 out.ufunc.maximum(lam, out=out)
 
                 # Global scaling
-                out /= lam
+                out /= (lam)
 
                 # Pointwise division
                 out.divide(diff, out)
 
     return _ProximalConvConjL1
 
 
-# TODO: move notes to ODL doc
+def proximal_l1(space, lam=1, g=None):
+    """Proximal operator factory of the l1-semi-norm.
+
+    Function for the proximal operator of the functional F where F is an
+    l1-semi-norm
+
+        F(x) = lam || ||x-g||_p ||_1
+
+    with x and g elements in ``space``, scaling factor lam, and point-wise
+    magnitude ||x||_p of x. If x is vector-valued, ||x||_p is the point-wise
```

Comment:
I made a commit with `isotropic` now, perhaps have a look at it and consider it. Looks good for the examples we have, but I'll have to admit it is unsatisfactory mathematically.